src/main/kotlin/org/appjam/smashing/domain/auth/social/kakao/KakaoAuthTokenValidator.kt

```
@@ -0,0 +1,27 @@
+class KakaoAuthTokenValidator(
+    private val kakaoApiClient: KakaoApiClient
+) {
+    fun extractKakaoId(authAccessToken: String): String =
+        try {
+            val token = if (authAccessToken.startsWith(PREFIX)) authAccessToken else "$PREFIX$authAccessToken"
+
+            val response = kakaoApiClient.getUserInfo(token)
+
+            response.id.toString()
+
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)